components/automate-ui/src/app/pages/+compliance/shared/reporting/stats.service.ts

```diff
@@ -183,17 +193,26 @@ export class StatsService {
   }
 
   getReports(reportQuery: ReportQuery, listParams: any): Observable<any> {
+      return this.getReportsWithPages(reportQuery, listParams, 1, 10).pipe(
+        map(reportCollection => reportCollection.reports));
+  }
+
+  getReportsWithPages(reportQuery: ReportQuery, listParams: any,
```

Comment:
Ooh, there's that `any` again. Sorry, I just have this visceral response to it whenever I see it being _added_.